eras/conway/impl/src/Cardano/Ledger/Conway/Governance/DRepPulser.hs

```diff
@@ -320,11 +365,12 @@ finishDRepPulser (DRPulsing (DRepPulser {..})) =
   (PulsingSnapshot dpProposals finalDRepDistr dpDRepState, ratifyState')
   where
     !leftOver = Map.drop dpIndex $ umElems dpUMap
-    !finalDRepDistr = computeDRepDistr dpStakeDistr dpDRepState dpProposalDeposits dpDRepDistr leftOver
+    (!finalDRepDistr, !finalStakePoolDistr) =
```

Comment:
Those bangs aren't really necessary, since `ratifyEnv` will force them, but if you do want to add them then you also need to add a bang on the let binding itself, since those are lazy.
```suggestion
    !(!finalDRepDistr, !finalStakePoolDistr) =
```